weblab/experiments/views.py

```diff
@@ -170,6 +170,93 @@ class ExperimentVersionListView(VisibilityMixin, DetailView):
     template_name = 'experiments/experiment_versions.html'
 
 
+class ExperimentComparisonView(TemplateView):
+    """
+    Compare multiple experiment versions
+    """
+    template_name = 'experiments/experimentversion_compare.html'
+
+    def get_context_data(self, **kwargs):
+        versions = ExperimentVersion.objects.filter(
+            pk__in=map(int, self.kwargs['version_pks'][1:].split('/'))
```

Comment:
Should we give some sort of error if the user requests non-existent versions, or versions they're not allowed to view? Or just silently filter so it only shows ones they can see?

I think my preferred option would be to compare the versions they can see (if any) but add an error notification "Some requested experiment results could not be found (or you don't have permission to see them)"